astropy/time/formats.py

```diff
@@ -72,6 +73,11 @@ def _regexify_subfmts(subfmts):
     return tuple(new_subfmts)
 
 
+class TimeMaskedArray(np.ma.MaskedArray):
```

Comment:
But I think that's OK. Right now, if I don't mask anything, I get:
```
In [23]: Time(np.arange(50000, 50010), format='mjd').cxcsec
Out[23]: 
array([-70329538.816, -70243138.816, -70156738.816, -70070338.816,
       -69983938.816, -69897538.816, -69811138.816, -69724738.816,
       -69638338.816, -69551938.816])
```
so why shouldn't a masked `Time` return a `MaskedArray`?